src/main/scala/com/fluidcode/processing/bronze/BronzeLayer.scala

```diff
@@ -11,7 +12,7 @@ class BronzeLayer(conf: Configuration, spark: SparkSession, path: String) {
     val bronzeData = spark.readStream.schema(bronzeSchema).json(path)
 
     bronzeData.writeStream
-      .option("checkpointLocation", s"${conf.checkpointDir}")
+      .option("checkpointLocation", s"${conf.rootPath}/${conf.database}/${conf.bronzeTable}/$CHECKPOINT_DIR")
```

Comment:
same problem remains. start from the checkpoint directory and then specify the table.